sBTC SIP

```diff
@@ -0,0 +1,166 @@
+# sBTC SIP
+
+## SIP Number: 022
+**Title:** A Decentralized Two-Way Bitcoin Peg  
+**Type:** Standard  
+**Status:** Draft  
+
+## Abstract
+sBTC is a novel digital asset that lets you move Bitcoin in and out of the Stacks blockchain. With sBTC, users can interact with Clarity smart contracts, which enable Bitcoin applications such as payments, decentralized lending, decentralized exchanges, and BTC-backed stablecoins.
+
+sBTC is a SIP-010 token on the Stacks blockchain, backed 1:1 against BTC, and operated by a decentralized set of signers. When BTC is locked on the Bitcoin L1, an equivalent amount of sBTC is issued on the Stacks layer, ensuring a consistent 1:1 ratio of sBTC:BTC. Users can redeem their sBTC at any time by submitting a withdrawal request. Once the request is processed by sBTC signers, BTC is returned to the user’s specified address on the Bitcoin L1.
+
+sBTC will have a crucial role in scaling Bitcoin, as well as introducing new and innovative functionalities to users, growing the size of the overall Bitcoin ecosystem. This SIP aims to describe the sBTC system, the process for signer selection, and the features available in the initial release compared to subsequent versions of sBTC. It does not attempt to describe the low-level technical details of any subsequent release, which will be provided in a future SIP.
```

Comment:
"sBTC will have a crucial role in scaling Bitcoin" --> Stacks doesn't scale either ([no blockchain scales](https://gist.github.com/jcnelson/edc40f883b9105aac4e04420cc4d536a).  I wouldn't focus on scalability as a key result, since it's untrue.  Perhaps focus instead on expressiveness offered by Clarity over BTC script?

Reply:
Possibly i would focus more on the use-case for sBTC - unlocking bitcoin for use in dapps and the like: "unlock the bitcoin economy" "bitcoin as a programmable asset" as it were.  Agreed with jude about the scalability wording, technically that's not something that we'll see (we're still bound by the limitations of a blockchain), even though it *appear* that way from a distance.